### ISSUE TITLE
Support Web Audio

### ISSUE THREAD

**Issue by bhackett1024** [replayio/gecko-dev#55]
The Web Audio API cannot currently be used when recording/replaying.

https://app.intercom.com/a/apps/k7f741xx/inbox/inbox/4328427/conversations/165083900017962

**Comment by phishy** [comment]
Yeah this is a bummer. We don't necessarily need to hear the audio, it would just be nice if the page would render. I also don't know how to catch the Exception, because Replay wont render the errored recording so I can see the console.

**Comment by jasonLaster** [comment]
Hey @phishy that's a great point. There might be some things we can do on the "do not throw" side before actually supporting audio. CC @bhackett1024

**Comment by jasonLaster** [comment]
@phishy out of curiosity, what APIs are you using?
https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API

**Comment by phishy** [comment]
AudioContext with peaks.js

**Comment by gideonred** [comment]
This ticket has been moved to [RUN-179](https://linear.app/replay/issue/RUN-179/support-web-audio)

**Comment by phishy** [comment]
Cool. So now it's private and nobody can see the status. That's fun.

**Comment by gideonred** [comment]
This was closed by mistake. Re-opening.

**Comment by ceceliacreates** [comment]
One potential workaround suggested is to expose an environment variable option to bypass or allow for custom handling of this error when recording. cc @gideonred
